schemas/user.py

```
@@ -0,0 +1,103 @@
+    password: str
+
+class UserUpdateRequest(BaseModel):
+    nickname: str | None = Field(None,min_length=2,max_length=20,pattern=r"^[가-힣a-zA-Z0-9]+$",description="한글,영문,대소문자만 허용한다.")
+    password: str | None = Field(None,min_length=8,max_length=20,description="바꾸고 싶은 비밀번호")
+    current_password: str = Field(...,min_length=8,max_length=20,description="원래 사용하고 있던 비밀번호")
+
+    # @field_validator에 경우 필드 검사 (닉네임이 2글자 이상인가?)
+    # @model_validator에 경우 여러 필드 사이의 관계를 검증할 때 사용한다.
+    @field_validator("password")
+    @classmethod
+    def validate_password_complexity(cls, v: Optional[str]):
```

Ruff (0.14.11)

Missing return type annotation for classmethod `validate_password_complexity`

(ANN206)